engine/gamesys/src/gamesys/resources/res_texture.cpp

```diff
@@ -353,7 +355,7 @@ namespace dmGameSystem
         {
             TextureResource* texture_res = new TextureResource();
             texture_res->m_Texture = texture;
-            params.m_Resource->m_Resource = (void*) texture_res;
+            params.m_Resource->m_Resource = texture_res;
```

Comment:
Huh, I don't see a change here. I expected _something_ to have changed, since the type of m_Texture has changed?
Are we assigning a HTexture to the HAssetHandle ?
Perhaps I'm not used to having handles inherit from each other (I think Windows.h does it a lot though, so I guess it's common).